pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -824,6 +824,10 @@ cdef class _Timedelta(timedelta):
         """ Returns a numpy.timedelta64 object with 'ns' precision """
         return np.timedelta64(self.value, 'ns')
 
+    def to_numpy(self, dtype=None, copy=False):
```

Comment:
Why copy and dtype kwargs?

Reply:
This came from the discussion in https://github.com/pandas-dev/pandas/issues/24653.  I suppose the question is whether we need this to behave like `to_numpy` does for non-scalars in the sense that it accepts the arguments and doesn't raise an exception.  If not, then we can either leave them out all together, or keep them in to avoid the `TypeError` and raise another, or issue a warning.

Reply:
these args are for compatibility and need to be here (maybe mention that in the doc string);

Reply:
maybe warn/raise to prevent these from being silently ignored if a user tries to use them?

Reply:
Though it seems like copy is harmless I can see how this would make sense for dtype, where one could specify a type and wonder why they're not getting that type back.

Reply:
Should we add a warning here or leave it?

Reply:
warnings are not needed